trin-types/src/cli.rs

```diff
@@ -129,13 +129,13 @@ pub struct TrinConfig {
     )]
     pub networks: Vec<String>,
 
-    /// Number of Kilobytes to store in the DB
+    /// Storage capacity specified in megabytes.
     #[structopt(
-        default_value(DEFAULT_STORAGE_CAPACITY),
+        default_value(DEFAULT_STORAGE_CAPACITY_MB),
         long,
-        help = "Maximum number of kilobytes of total data to store in the DB"
+        help = "Maximum number of megabytes of total data to store in the DB"
```

Comment:
Might be nice to clarify to users something like:
```suggestion
        help = "Maximum number of megabytes of total data to store in the DB. The actual disk size will exceed the limit, because of overhead."
```